LDRH_choice.py

```diff
@@ -231,7 +247,14 @@ def importConditions(path):
         'Spatial': data.StairHandler(startVal = 150,
             stepType = 'db', stepSizes=[3,3,2,2,1,1],#[8,4,4,2,2,1,1], #reduce step size every two reversals
             minVal=0, maxVal=350, nUp=1, nDown=3,  #will home in on the 80% threshold
-            nTrials = 10)
+            nTrials = 10),
+        'posterior_matching': {
```

Comment:
I did this:

if use_posterior_matching==False:
     all_handlers['staircasing']={...}
     this_handler = all_handlers['staircasing']
elif use_posterior_matching==True:
     all_handlers['posterior_matching']={...}
     this_handler = all_handlers['posterior_matching']

then, below, I just do 
if operation:
    handler = this_handler[task][operation]
else:
    handler = this_handler[task]